### PR TITLE
Add Exercise 13 files

### PR DESCRIPTION
**Pull request by ElizaAlimasa**
Please review my file, Thank you!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,226 @@
...
+    <hr />
+    <div>
+      <h2 class="remarks">Comment Section</h2>
+      <hr />
+      <div class="content">
+        <div>
+          <button onclick="sortComments('asc')">Oldest</button>
+          <button onclick="sortComments('desc')">Newest</button>
+        </div>
+        <ul class="com-li" id="commentList"></ul>
```
Comment (Tapispisan-YvesJohn) [inline index.html]: IDs should be written in snake case. All lower case separated by underscore. 
See the reference for clarifications. 
```suggestion
        <ul class="com-li" id="comment_list"></ul>
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: index.html [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+        <div class="container">
+          <p>Hello Kitty</p>
+        </div>
+      </div>
+    </div>
+    <hr />
+    <div class="details">
+      <h2 class="contacts">Contact Me</h2>
+      <hr />
+      <form id="commentForm">
```
Comment (Tapispisan-YvesJohn) [inline index.html]: IDs should be written in snake case. All lower case separated by underscore. 
See the reference for clarifications. 
```suggestion
      <form id="comment_form">
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

File: index.html [hunk 3]
```diff
@@ -0,0 +1,226 @@
...
+        <div class="email">
+          <label for="email">Email:</label>
+          <input type="email" id="email" name="email" required />
+        </div>
+        <div class="fullname">
+          <label for="fullname">Full Name:</label>
+          <input type="text" id="fullname" name="fullname" required />
```
Comment (Tapispisan-YvesJohn) [inline index.html]: IDs should be written in snake case. All lower case separated by underscore. 
See the reference for clarifications. 
```suggestion
          <input type="text" id="full_name" name="fullname" required />
```

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

**Review by Tapispisan-YvesJohn — CHANGES_REQUESTED** [review]
Please see the suggested changes and apply it properly. Also check your script.js and style.css to properly reflect the changes in your html file.

**Comment by ElizaAlimasa** [comment]
Hello. I've made some changes and updated the file. Please review my code again, thank you!

File: index.html [hunk 4]
```diff
@@ -0,0 +1,226 @@
...
+    <hr />
+    <div class="details">
+      <h2 class="contacts">Contact Me</h2>
+      <hr />
+      <form id="comment_form">
+        <div class="email">
+          <label for="email">Email:</label>
+          <input type="email" id="email" name="email" required />
+        </div>
+        <div class="fullname">
```
Comment (Tapispisan-YvesJohn) [inline index.html]: I apologize I missed this part. Class names should be written in kebab case. All lower case separated by hyphen. See the see the suggested code or the reference for clarifications. 
```suggestion
        <div class="full-name">
```
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5

**Review by Tapispisan-YvesJohn — CHANGES_REQUESTED** [review]
Adjust your js code and css styling according to the suggested changes too.

**Comment by ElizaAlimasa** [comment]
Hello, I have implemented the suggested changes in the files. Please review my files again, Thank you!

**Review by Tapispisan-YvesJohn — APPROVED** [review]
Great job! You can merge it now.

**Comment by ElizaAlimasa** [comment]
Thank you!

**Pull request merged**